bin/i18n/i18n_metrics.rb

```diff
@@ -0,0 +1,37 @@
+require 'cdo/aws/metrics'
+require 'aws-sdk-ec2'
+require 'net/http'
+require 'socket'
+
+module I18n
+  module Metrics
+    I18N_METRICS_NAMESPACE = 'I18n'.freeze
+
+    def self.report_runtime(addtl_dimensions)
+      log_metric(:Runtime, addtl_dimensions, yield)
+    end
+
+    def self.machine_id
+      @machine_id ||= begin
+        Net::HTTP.get(URI.parse(CDO.ec2_metadata_endpoint + 'instance-id'))
```

Comment:
I would recommend moving `ec2_metadata_endpoint` with `instance-id` as `ec2_instance_id_endpoint: 'http://169.254.169.254/latest/meta-data/instance-id'`.
It would allow us easily change this config if someday we decide to separate the endpoint to get the `instance-id`.